### PR TITLE
I made some new sprites for Kidan.

### PR DESCRIPTION
**Pull request by TravellingMerchant**
I made some new sprites for the Kidan race. They no longer look like a
stick insect.

Support for them has been massive on discord and the forums. Gibs and
body parts all work fine. No clipping issues with clothes beyond the
antenna that the original Kidan have.

Eye colour is also customisable. Added that into the station.dm

Special thanks to TastyFish for teaching me the .dmi files and a bunch of other stuff!

:cl:
imageadd: Kidan have new sprites!
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 1]
```diff
@@ -350,6 +350,7 @@
 	flags = IS_WHITELISTED
 	clothing_flags = HAS_SOCKS
 	bodyflags = FEET_CLAWS
+	eyes = "kidan_eyes"
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/species/station.dm]: huh this is a thing

**Comment by Fox-McCloud** [comment]
Needs pics.

File: code/modules/mob/living/carbon/human/species/station.dm (hunk 1 above)
Comment (Tastyfish) [inline code/modules/mob/living/carbon/human/species/station.dm]: Yeah, I didn't know either until I scrambled through the code and found it.

**Comment by ProperPants** [comment]
@Fox-McCloud 
ss 2016-03-16 at 05 25 27 ss 2016-03-16 at 04 54 55 ss 2016-03-16 at 04 52 43
ss 2016-03-16 at 04 51 05 ss 2016-03-16 at 04 53 16 ss 2016-03-16 at 04 53 53

**Comment by TravellingMerchant** [comment]
Ah right, I assumed you would be able to see in the files. Here you go, full sheet of the sprites. Ignore the bottom one with wings as they were not included in the pull request. I may think about adding them at a later point in time.

http://puu.sh/nFT2k.png

As for the robe clipping, looks pretty cool actually. Its no worse than Vox or Dionas clipping, which is horrible.

**Comment by FalseIncarnate** [comment]
I'm fine with the bit of clipping at the shoulders, I was just worried their arms and such would bulge out of every jumpsuit.

They definitely look more intimidating than old kidan, which is a major :+1: in my opinion. Assuming the winged sprites aren't currently select-able right now based on how few lines of code were adjusted?

**Comment by TravellingMerchant** [comment]
I'm not sure I understand how to do that. In fact I don't understand it at all. Does this mean this cant be added without one? 

Wings were not included in this, no.

**Comment by Nienhaus** [comment]
They look fine @FalseIncarnate 
jkavfz9
Same clipping as the robes.

**Comment by Spacemanspark** [comment]
:+1:

**Comment by FalseIncarnate** [comment]
Yea, I'm aware Nein (though thanks for the picture for everyone else). I'm fine with the shoulder bit, I was just mentioning that my initial worry was the arms and such (as I had voiced in the forum thread for this).

**Comment by Nienhaus** [comment]
Ah okay. I believe I misunderstood what you meant.

**Comment by TravellingMerchant** [comment]
So, good enough to merge? :+1:

**Comment by KasparoVy** [comment]
Decided to axe the Kidan female vestigal wings at the last minute? Can't seem to find them in the actual .dmi files

**Comment by TravellingMerchant** [comment]
Aye, axed them due to clipping on clothes. May re-add them later if I add any shell/horn styles as customisable options. I just want to focus on getting these in the game first.

**Comment by KasparoVy** [comment]
Good idea, wings or whatever as a body accessory might work out the way you intend

**Comment by TheDZD** [comment]
Might want to take a closer look at those sprites, I think a bug crawled onto some of them.

**Pull request merged**